shared/hybrid/AppDelegate+SalesforceHybridSDK.m

```diff
@@ -95,7 +95,9 @@ - (BOOL)sfsdk_swizzled_application:(UIApplication *)application didFinishLaunchi
     
     [self initializeAppViewState];
     [[SalesforceSDKManager sharedManager] launch];
-    return [self sfsdk_swizzled_application:application didFinishLaunchingWithOptions:launchOptions];
+    return YES; // we don't want to run's Cordova didFinishLaunchingWithOptions - it creates another window with a webview
```

Comment:
We used to return yes up until a couple of months back. However sample apps continue working because their AppDelegate.m was doing less than the AppDelegate.m that you actually get when doing cordova create (see below).